shoop/core/management/commands/shoop_makemessages.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Extract translatable strings from shoop
+"""
+from __future__ import unicode_literals
+
+import logging
+import subprocess
+
+import os
+from django.conf import settings
+from django.core import management
+from django.core.management import BaseCommand
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Command(BaseCommand):
```

Comment:
IMHO this exactly the reason why it should be a management command provided by Shoop Core. Being installed with Shoop means it is available for addon developers very easily.

BUT:
* The shoop_makemessages command needs to be adapted for collecting messages from any given directory (not just shoop as it does now).
* Same for the "setup.py build_translations" command, so that addons could `import shoop_setuputils` in their setup.py and use it too.